mirrord/config/src/feature/network/incoming.rs

```diff
@@ -385,6 +386,12 @@ pub enum IncomingMode {
     /// data on a port is HTTP (in a best-effort kind of way, not guaranteed to be HTTP), and
     /// steals the traffic on the port if it is HTTP;
     Steal,
+
+    /// <!--${internal}-->
```

Comment:
Don't we want this to be available to users?

Reply:
See the other modes, they all have it so I assumed that's on purpose.